testprojects/src/js/hello_react/src/test/index.test.js

```diff
@@ -0,0 +1,16 @@
+// Copyright 2023 Pants project contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+/**
+ * @jest-environment jsdom
+ */
+
+import React from "react";
+import Server from "react-dom/server";
+
+import HelloPage from "../App.jsx";
+
+it("page to greet you", () => {
+  const rendered = Server.renderToString(<HelloPage />);
```

Comment:
Given this uses JSX syntax, should this be named `index.test.jsx` (not `.js`)?

Reply:
Yes, which also reveals that I completely missed adding targets for jsx test files :roll_eyes: